app/controllers/missing.php

```diff
@@ -3,6 +3,11 @@
 class missing extends Controller {
 
     function __construct() {
+        $this->load_library('deployment_settings', 'settings');
+        if ($this->settings->missing_person() != 1) {
```

Comment:
Why are you comparing it with `1`? Wouldn't simply `!$this->settings->missing_person()` be better?